test/vanilla/Configurations/hidden-methods.md

```diff
@@ -0,0 +1,24 @@
+# Hidden Methods
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+``` yaml
+base-folder: ../../../
+input-file: node_modules/@microsoft.azure/autorest.testserver/swagger/body-complex.json
+payload-flattening-threshold: 1
+clear-output-folder: true
+directive:
+  - from: code-model-v1
+    where: $.operations[*].methods[?(@.serializedName == 'basic_putValid')]
+    transform: >-
+      $.hidden = true;
+      $.excludeFromInterface = true;
```

Comment:
The fact that you added the member to the language-specific implementation `Method` (ie, `MethodCs.cs` ) and then simply used a `directive: transform`  to enable it is exactly how we wanted to approach this type of feature.

![image](https://user-images.githubusercontent.com/133736/31189935-7e48f2b6-a8ee-11e7-839d-db1d0dfc06f9.png)

Reply:
Excellent! I'm glad this was the right approach.